source/extensions/load_balancing_policies/random/BUILD

```diff
@@ -12,6 +12,10 @@ envoy_cc_extension(
     name = "config",
     srcs = ["config.cc"],
     hdrs = ["config.h"],
+    extra_visibility = [
+        # previously considered core code.
+        "//test:__subpackages__",
```

Comment:
optional but I'd super appreciate a follow-up moving some of the tests for the infrequently used LB to the individual directories.  Obviously not worth it for round robin but might be doable for lesser used ones.